livekit_egress.proto

```diff
@@ -330,6 +345,15 @@ message SegmentsInfo {
   int64 ended_at = 7;
 }
 
+message AutoParticipantEgress {
+  oneof options {
+    EncodingOptionsPreset preset = 1; // (default H264_720P_30)
+    EncodingOptions advanced = 2;     // (optional)
+  }
+  repeated EncodedFileOutput file_outputs = 3;
+  repeated SegmentedFileOutput segment_outputs = 4;
```

Comment:
would be good to have a `repeated TrackSource sources` here to filter the sources they want.. by default it's all.

Reply:
I agree it's a good feature to have, but any reason not to add it shortly after the 1st release to avoid creeping the initial scope?

Reply:
yeah that'd be fine too. though if it's one of those things that's simple to design in, then that'd be preferred. I think its utility would be severely limited if it could only handle a participant's camera tracks